tests/coloring/test_coloring.py

```diff
@@ -65,7 +64,7 @@ def test_solvers():
     assert coloring_model.graph.nodes_name is not None
     solvers = solvers_map.keys()
     for s in solvers:
-        if s == ColoringLP and not gurobi_available:
+        if s == ColoringLP:
```

Comment:
Why do we skip this solver evevn when shipping gurobi now? It was supposed to work up to now.
(And it will decrease the coverage i suppose)